extensions/wikia/Recirculation/js/helpers/CakeRelatedContentHelper.js

```diff
@@ -42,10 +42,14 @@ define('ext.wikia.recirculation.helpers.cakeRelatedContent', [
 
     function formatData(data) {
         data.items = data.items.map(function(item) {
-            item.title = item.title.split(' | ')[0];
-            return item;
+			if (item.source  == "discussions") {
+				item.meta.authorAvatarUrl = item.author.avatar.src;
+			}
+			item.author = item.author.username;
+			item.title = item.title.split(' | ')[0];
+			return item;
         }).sort(utils.sortThumbnails);
-
+		debugger
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this debugger statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ADebuggerStatement)